src/content/docs/en/glossary.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Glossary 
+description: A collection of terms and definitions used in the Astro documentation and community.
+i18nReady: false
+---
+
+This glossary is intended to explain the terminology used in both the Astro documentation and in the Astro community.
+
+{ /* // Uncomment this section when we have more terms to add to the glossary.
+It is separated into Astro-specific terms and general web development terms.
+## Astro-Specific Terms 
+*/}
```

Comment:
I believe it can help to know what type of general web dev terms are acceptable here.
Best I can think of are:  CLI, Editor, Extensions, Frameworks.

Reply:
I will add a couple more guidelines here. I am not sure whether or not the glossary will broaden in scope in the future. However, it wouldn't hurt to keep it in the mix!

Reply:
sounds good 👍 as long as Sarah agrees I agree 👍

Reply:
```suggestion

{ /* // Uncomment this section when we have more terms to add to the glossary.
It is separated into Astro-specific terms and general web development terms.
## Astro-Specific Terms 

Some future guidelines for this:
- Astro-Specific: Content Collections, ViewTransitions, Astro Component
- General: Editor, LSP, Front-End Library, Framework, Metaframework, Framework Component
*/}
```

How is this?